src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -1,5 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Advancement_Namespace__c</fullName>
+        <defaultValue>&quot;adv1&quot;</defaultValue>
+        <description>The namespace of the Gift Processing package.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The namespace of the Gift Processing package.</inlineHelpText>
```

Comment:
Is the name "Gift Processing" final?

Reply:
Define "final" ;-).  But yeah, that's what it's being called now.

Reply:
Hi @smarovich, this custom setting field is being added to our product to enable internal developers. Customers never need to understand what it is, and they can't use it for anything. I'm thinking:

"System use only: namespace for cross package functionality"

Thoughts?

Reply:
Looks good, @njjc. Some minor edits: "For system use only: Namespace for cross package functionality."